apps/web/src/pages/Profile/CareerTimelineAndRecruitmentPage/constants.ts

```diff
@@ -10,9 +10,9 @@ export type PageSectionId = ObjectValues<typeof PAGE_SECTION_ID>;
 
 export const titles = defineMessages({
   careerTimelineAndRecruitment: {
-    defaultMessage: "Career timeline and recruitment",
-    id: "Icl1fF",
-    description: "Name of Career timeline and recruitment page",
+    defaultMessage: "Career timeline",
+    id: "TUfJUD",
+    description: "Name of Career timeline page",
```

Comment:
Are we planning on moving the qualified recruitments to a different section? Or do we consider them part of the entire "career timeline" @substrae ? I may just not understand what a qualified recruitment is but as a user, I don't think I'd expect to find them on a page titled "Career timeline". Though, that is also true for education and community experience, etc 😕

Reply:
@esizer 

There is a plan to rework this page in its entirety, and qualified recruitments are indeed being given their own page in time. That said, they currently exist on the resume page because they are a career achievement of sorts, and there is no other context that suits them for now. In terms of the relabelling of the section, the dashboard provides a link to the recruitments right there in the card, among other paths including from application cards, confirmation alerts when you are entered into a pool, etc., so I'm not worried about the navigation right now.

Reply:
No change required at this time

Reply:
Please do let us know when this might change so that we can ensure the copy editing pipeline can treat it.